ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/SimulateV1Parameter.java

```diff
@@ -28,7 +28,8 @@ public SimulateV1Parameter(
       @JsonProperty("blockStateCalls") final List<JsonBlockStateCallParameter> blockStateCalls,
       @JsonProperty("validation") final boolean validation,
       @JsonProperty("traceTransfers") final boolean traceTransfers,
-      @JsonProperty("returnFullTransactions") final boolean returnFullTransactions) {
-    super(blockStateCalls, validation, traceTransfers, returnFullTransactions);
+      @JsonProperty("returnFullTransactions") final boolean returnFullTransactions,
```

Comment:
what do you mean by fullTransactions ?

Reply:
this is just formatting in this PR - @Gabriel-Trintinalia added the returnFullTransactions option to the spec afaict - maybe [metamask](https://docs.metamask.io/services/reference/ethereum/json-rpc-methods/eth_simulatev1/) asked for it?

this defaults to false, so only transaction hashes are returned rather than the full transaction bodies.